drivers/eeprom/eeprom_at2x.c

```diff
@@ -35,17 +35,26 @@ LOG_MODULE_REGISTER(eeprom_at2x);
 #define EEPROM_AT25_STATUS_BP1 BIT(3) /* Block Protection 1 (RW) */
 
 #define HAS_WP_OR(id) DT_NODE_HAS_PROP(id, wp_gpios) ||
-#define ANY_INST_HAS_WP_GPIOS (DT_FOREACH_STATUS_OKAY(atmel_at24, HAS_WP_OR) \
-			       DT_FOREACH_STATUS_OKAY(atmel_at25, HAS_WP_OR) 0)
+#define ANY_INST_HAS_WP_GPIOS                                                                      \
+	(DT_FOREACH_STATUS_OKAY(atmel_at24, HAS_WP_OR)                                             \
+		 DT_FOREACH_STATUS_OKAY(atmel_at25, HAS_WP_OR) 0)
```

Comment:
This now also has to check `st_m24xxx` compatibles?

Reply:
true that